client/environment_import.go

```diff
@@ -0,0 +1,96 @@
+package client
+
+type EnvironmentImport struct {
+	Id             string     `json:"id"`
+	Name           string     `json:"name,omitempty"`
+	OrganizationId string     `json:"organizationId"`
+	Workspace      string     `json:"workspace,omitempty"`
+	Variables      []Variable `json:"variables,omitempty"`
+	GitConfig      GitConfig  `json:"gitConfig,omitempty"`
+	IacType        string     `json:"iacType,omitempty"` // opentofu or terraform
+	IacVersion     string     `json:"iacVersion,omitempty"`
+}
+
+type Variable struct {
+	Name        string `json:"name"`
+	Value       string `json:"value,omitempty"`
```

Comment:
is omitempty needed here?

Reply:
All of what this resource does is create an entity that we'll use to auto fill a form in the UI
So I want them to be able to just specify variables without values ( and leave their value as empty in the UI for them to manually fill in )

This is very unlikely though I can remove it, its not important